wiki/en/Running-a-Server.md

```diff
@@ -298,6 +297,32 @@ You yourself should connect using the local network (LAN) address of the machine
 
 Most domestic internet connections change their external IP address after a short period. To avoid problems with this, you might want to set up "dynamic DNS" to get a static (sub-)domain you can share with others. Please research how to do that for your specic set up. Your router might support some "dynamic DNS" providers out of the box. If this is not the case, set up a dynamic DNS client as described by the dynamic DNS provider you chose.
```

Comment:
```suggestion
Most internet connections change their external IP address after a short period. To avoid problems with this, you might want to set up "dynamic DNS" to get a static (sub-)domain you can share with others. Please research how to do that for your particular situation. Your router might provide support for some "dynamic DNS" providers. If this is not the case, set up a dynamic DNS client by following instructions provided by the dynamic DNS provider you chose.
```

Reply:
I'm not sure how common it is to _force_ a change.  Maybe "Some" would be better than "Most"?  Or the more technically accurate: "Unless you have agreed in your ISP contract to have a fixed IP address, then you may find that your address changes over time."

Reply:
Since this isn't part of this PR, but is [part of mine](https://github.com/jamulussoftware/jamuluswebsite/pull/875), I can update it there to avoid conflicts.